templates/scheduler/scheduler-deployment.yaml

```diff
@@ -32,6 +32,10 @@ spec:
   serviceName: {{ .Release.Name }}-scheduler
 {{- end }}
   replicas: 1
+  strategy:
+    type: RollingUpdate
```

Comment:
The type here needs to be `Recreate`, not rolling update. 

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec